src/app/rosetta/lib/block.ml

```diff
@@ -85,14 +85,13 @@ module Internal_command_info = struct
       match t.kind with
       | `Coinbase ->
           (* The coinbase transaction is really incrementing by the coinbase
-         * amount and then decrementing by the fees paid. *)
-          [ {Op.label= `Coinbase_inc; related_to= None}
-          ; {Op.label= `Fee_payer_dec; related_to= Some `Coinbase_inc} ]
```

Comment:
Do you not have to track the prover fee that gets deducted from the coinbase amount or is that part of list of fee transfers?

Reply:
it is part of the fee transfer list (I think). Anyway, getting the rosetta validation suite to pass will uncover if this is true.